src/Log.h

```diff
@@ -771,8 +771,13 @@ class Log {
 
                 if (ok && p_UseSpecifiedValue && (thisProperty == p_SpecifiedProperty)) {                                       // replace specified property?
                     value    = p_SpecifiedPropertyValue;                                                                        // yes - use value passed as parameter
-                    valueStr = boost::apply_visitor(FormatVariantValue(), value, fmtStr);                                       // format value
-                    logRecord += valueStr + delimiter;                                                                          // add value string to log record - with delimiter
+                    if (hdf5) {                                                                                             // yes - HDF5 file?
```

Comment:
Not sure how much you care about these sorts of things but the comments here aren't aligned properly

Reply:
I do care :-)  Fixed.  Good pickup - thanks